pkg/cli/project/project.go

```diff
@@ -158,14 +158,17 @@ func (o ProjectOptions) Run() error {
 		currentProject = currentContext.Namespace
 	}
 
+	client, kubeclient, err := o.ClientFn()
+	if err != nil {
+		return err
+	}
+	if err := client.RESTClient().Get().Resource("projectrequests").Do(context.TODO()).Into(&metav1.Status{}); err != nil {
+		fmt.Fprintf(o.Out, "%v\n", err)
+		return nil
```

Comment:
If this is error, just `return err` is the best option.

Reply:
updated, fixed

Reply:
but this  changes behavior of `oc project` to return an error.  
As/is, if user w/out self-provisioner role runs `oc project` they'll see this:
```console
$ oc project
No project has been set. Pass a project name to make that the default.
```
and now, with this PR, it will be this:
```console
$ oc project
Error from server (Forbidden): This is a friendly message...
```
or, if no projectRequestMessageSet, this:
```console
$ oc project
Error from server (Forbidden): You may not request a new project via this API.
```
(this message without the set projRequestMessage is now consistent with the error a user gets with `oc new-project foo` now, which makes more sense to me)